### ISSUE TITLE
Simultaneous vpn usage

### ISSUE THREAD

**Issue by osrf-migration** [osrf/cloudsim-legacy#61]
**Original report ([archived issue](https://osrf-migration.github.io/osrf-archived-gh-pages/#!/osrf/cloudsim-legacy/issues/61)) by michele hallak (Bitbucket: [mhallak](https://bitbucket.org/%7Bf8cca72b-2f8a-4372-bcb5-10ea801a3fe9%7D/)).**

----------------------------------------
It seems that two people cannot connect to the field computer via the router at the same time (from two different stations): each time one logs in, the second is logged out (via SSH)
The problem might be caused by the fact that everyone is using the same keys.
Everyone is using the same keys because of the problem [#60](https://osrf-migration.github.io/osrf-archived-gh-pages/#!/osrf/cloudsim/issues/60/console-doesnt-display-constellation) (osrf/cloudsim#60). Nobody sees constellation so they cannot download SSH keys.

**Comment by osrf-migration** [comment]
**Original comment by michele hallak (Bitbucket: [mhallak](https://bitbucket.org/%7Bf8cca72b-2f8a-4372-bcb5-10ea801a3fe9%7D/)).**

----------------------------------------
what's the point of having multiple users if they can work in parallel on the constellation?

**Comment by osrf-migration** [comment]
**Original comment by Carlos Agüero (Bitbucket: [caguero](https://bitbucket.org/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D/), GitHub: [caguero](https://github.com/caguero)).**

----------------------------------------

* changed kind from "bug" to "enhancement"

This is the expected behavior with the current OpenVPN configuration. Only a single client (OCU) can connect with the router at the same time. I have updated the issue as an enhancement and we will evaluate to change the vpn configuration after the VRC.

**Comment by osrf-migration** [comment]
**Original comment by Carlos Agüero (Bitbucket: [caguero](https://bitbucket.org/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D/), GitHub: [caguero](https://github.com/caguero)).**

----------------------------------------

* changed state from "new" to "on hold"

On hold until the end of VRC.

**Comment by osrf-migration** [comment]
**Original comment by Hugo Boyer (Bitbucket: [hugomatic](https://bitbucket.org/%7B3d87874e-7099-4981-b04f-57f0272faa7d%7D/), GitHub: [hugomatic](https://github.com/hugomatic)).**

----------------------------------------

* changed state from "on hold" to "open"
